src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar"
+      data-qa="big"
+    >
+      <input
+        class="input input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar"
+      data-qa="small"
+    >
```

Comment:
Similar to the previous form, this `<form>` tag should not be self-closed. Change `<form />` to `<form>` and ensure it has a corresponding `</form>` closing tag.